### ISSUE TITLE
Charger power reactive export/import have lost their state classes and units after update to v0.6.3

### ISSUE THREAD

**Issue by hummermann87** [lbbrhzn/ocpp#1449]
**Describe the bug**
Charger power reactive export/import have lost their state classes and units after update to v0.6.3.

**To Reproduce**
N/A

**Expected behavior**
No errors

**Screenshots**
![image](https://github.com/user-attachments/assets/5b0a3b92-3d0d-4f4a-a977-8ababd8cca4d)

**Desktop (please complete the following information):**
 - OS: Home Assistant, OS 14.1, Core 2024.12.5, Supervisor 2024.12.0

**Additional context**

**Comment by N3rdix** [comment]
Good catch, I missed that when introducing the reactive power device_class. I'll fix it asap

**Comment by BitsLeftOver** [comment]
Same problem.

**Comment by N3rdix** [comment]
Should be fixed once #1451 is included in an upcomibg Release
